src/Test.Utilities/CSharpCodeFixVerifier`2.cs

```diff
@@ -23,28 +24,39 @@ public static DiagnosticResult Diagnostic(string diagnosticId)
         public static DiagnosticResult Diagnostic(DiagnosticDescriptor descriptor)
             => CSharpCodeFixVerifier<TAnalyzer, TCodeFix, XUnitVerifier>.Diagnostic(descriptor);
 
-        public static async Task VerifyAnalyzerAsync(string source, params DiagnosticResult[] expected)
+        public static async Task VerifyAnalyzerAsync(string source, ReferenceAssemblies referenceAssemblies, params DiagnosticResult[] expected)
```

Comment:
@sharwell Can you take a look?

Reply:
This should not change. Tests that need a custom `ReferenceAssemblies` instance should use the full form:

```csharp
await new VerifyCS.Test
{
  ReferenceAssemblies = referenceAssemblies,
  ...
}.RunAsync();
```

Reply:
I expected this looking at this with fresh eyes today - i can move these helpers back into the test class tho, right? I assume the concern is with bloating the class with all kinds of unique use cases?